include/BSMPT/ThermalFunctions/NegativeBosonSpline.h

```diff
@@ -1022,4 +1025,686 @@ const double NegLinearInt[3001][2] = {
     {-2997, 203.7137417},   {-2998, 207.5373262},   {-2999, 211.3588207},
     {-3000, 215.1780691}};
 
+const std::vector<std::vector<double>> NegLinearIntTransposed = {
```

Comment:
Can you extract this in a cpp file? Like I did with the Kfunctions_grid_part1 and 2.

Compile time is increased by this and the readability is much better

Reply:
Could only fix this [partially](https://github.com/phbasler/BSMPT/commit/a0a407397222b51135e50fb3f5c0b8c41fde1b38). The problem with the NegLinearIntTransposed is that it is used to construct the spline "tk::spline JbosonNegativeSpline" so it has to be defined at compile time.
Any tips on how to solve this?

Reply:
It is still defined at compile time with the `extern` keyword. Or which part does complain if you seperate it like this?

Reply:
It is but 
```c++ 
tk::spline JbosonNegativeSpline(NegLinearIntTransposed[0],
                                NegLinearIntTransposed[1],
                                tk::spline::cspline,
                                false,
                                tk::spline::first_deriv,
                                -pow(M_PI, 2) / 12,
                                tk::spline::first_deriv,
                                3.816357189);
```
gives out a segmentation fault at compile time when we try to access ```NegLinearIntTransposed[0]``` and ```NegLinearIntTransposed[1]``` before being implemented.

Reply:
Alright. Regarding the time frame can you maybe spawn an issue for this and we can look into it later? This only affects compile time

Reply:
Sure. Issue created.